client/src/app/account-card/account-card.component.ts

```diff
@@ -13,18 +14,31 @@ export class AccountCardComponent implements OnInit {
 
   public onCardClick(): void {
     this.accountService.activateAccount(this.account._id);
+    this.accountService.setAccount(this.accountService.activeAccount);
     this.transactionService.getTransactions(this.account._id).subscribe({
```

Comment:
add pipe(UntilDestroyed(this)) for all places where you subscribe on observables

Reply:
for all places please